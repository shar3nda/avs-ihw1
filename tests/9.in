10
-2 7 4 7 9 9 3 2 8 2