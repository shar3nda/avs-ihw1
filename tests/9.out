-1 8 -1 1 3 6 6 8 9 3 6 -1 3 -1